docs/docs/glossary.md

```diff
@@ -0,0 +1,75 @@
+---
+id: glossary
+title: Glossary
+---
+
+### [Contestation Period](/head-protocol/terms/contestation-period) 
+The time window between closing the [head](../terms/hydra-head) and `fan out` allowed to `contest`.
```

Comment:
Does this mean we need to keep two explanations in sync? e.g. this one and `../terms/contestation-period.md`?

What's the reason of keeping it twice?

Reply:
this glossary.md is generated by the plugin.
i commited it to have the `last updated at` feature for this page working.

Reply:
We should `.gitignore` this one then.